### ISSUE TITLE
Particle data

### ISSUE THREAD

**Issue by anthonycmain** [VincentGarreau/particles.js#484]
Is there a way for me to use an array to inject data into the particles rather than random. 

I want to essentially put a different icon in each and then have a hover event which will show the title.

**Comment by matteobruni** [comment]
You can see this sample here: https://codepen.io/matteobruni/pen/YzpOZpq

It uses [tsParticles](https://github.com/matteobruni/tsparticles) for creating particles manually

**Comment by anthonycmain** [comment]
Whats the difference between this and tsParticles? Im a bit confused

**Comment by anthonycmain** [comment]
Can I put in more than just the position, e.g. I want to ideally put in a stroked circle with a different icon in for each particle

**Comment by matteobruni** [comment]
> Can I put in more than just the position, e.g. I want to ideally put in a stroked circle with a different icon in for each particle

Yes, you can see the properties here: https://particles.js.org/docs/classes/Options_Classes_ManualParticle.ManualParticle.html

You can specify `position` and `options`, the latest one contains the same object as the main `particles` object
